core/libraries/rest_api/CalculatedModelFields.php

```diff
@@ -82,6 +83,9 @@ protected function generateNewMapping()
                 'Registration' => array(
                     'datetime_checkin_stati' => $registration_class,
                 ),
+                'Attendee' => array(
+                    'userAvatar' => $attendee_class,
```

Comment:
this string `userAvatar` is what REST API clients will put in the querystring. The convention so far has been to have those be snake case. However, the method on the class can remain camelcase- `CalcualtedModelFields::retrieveCalculatedFieldValue()` takes care of translating the snake-case string into a camelcase one to determine which method to use.